### PR TITLE
Updating kop test to use multiple zones and HA

### PR DESCRIPTION
**Pull request by chrislovecnm**
Updated kubernetes_e2e.py not send in zones when the parameter
kops-multiple-zones is set to true. Updated kops canary test to create a
HA test across multiple zones with the latest etcd3.

### CODE REVIEW

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -693,6 +697,9 @@ def create_parser():
         '--kops-zones', help='Comma-separated list of zones else random choice')
     parser.add_argument(
         '--kops-build', action='store_true', help='If we need to build kops locally')
+    parser.add_argument(
+        '--kops-multiple-zones', default='false', help='Use multiple zones')
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: default is always false, do `action='store_true'` instead

File: scenarios/kubernetes_e2e.py [hunk 2]
```diff
@@ -345,9 +345,12 @@ def set_up_kops_aws(workspace, args, mode, cluster, runner_args):
     if args.aws_role_arn:
         profile = mode.add_aws_role(profile, args.aws_role_arn)
 
-    zones = args.kops_zones or random.choice(DEFAULT_AWS_ZONES)
-    regions = ','.join([zone[:-1] for zone in zones.split(',')])
-
+    zones = ""
+    regions = ""
+    if args.kops_multiple_zones is 'false'
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: `if args.kops_multiple_zones:`

File: scenarios/kubernetes_e2e.py [hunk 3]
```diff
@@ -364,6 +367,7 @@ def set_up_kops_aws(workspace, args, mode, cluster, runner_args):
         '--kops-nodes=%s' % args.kops_nodes,
         '--kops-ssh-key=%s' % aws_ssh,
         "--kops-ssh-user=admin",
+        "--kops-multiple-zones=%s" args.kops_multiple_zones,)
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: don't append if the flag is not set?

File: scenarios/kubernetes_e2e.py (hunk 2 above)
Comment (chrislovecnm) [reply scenarios/kubernetes_e2e.py]: it is a string ... not `False` ... does `not` work ... and I missed a colon ... I hate not being able to test this local

File: scenarios/kubernetes_e2e.py (hunk 3 above)
Comment (chrislovecnm) [reply scenarios/kubernetes_e2e.py]: It is set with a default in when I added the arg ...

**Comment by krzyzacy** [comment]
lol, ok that makes sense

**Comment by krzyzacy** [comment]
keep the args sorted :-)

**Pull request merged**